app/models/cart.rb

```diff
@@ -0,0 +1,5 @@
+class Cart < ApplicationRecord
+  has_many :cart_items, dependent: true
```

Comment:
```suggestion
  has_many :cart_items, dependent: :destroy
```
để destroy nhé. khi xóa cart thì cart_items cũng bị xóa luôn

Reply:
dạ vâng ạ